README.md

```diff
@@ -16,22 +16,28 @@
 ---
 <h3 align="center">Workflow of the Project</h3>
 
-📌 Gathering information about how to target audience. Analyze user needs and preferences. <br>
-📌 Developing a project plan that includes project scope, timelines, milestones,resources, and budget. <br>
-📌 Based on user research, define the key features . Identify the user interface design, navigation, and user experience requirements. <br>
-📌 Develop it using the appropriate programming languages and frameworks. <br>
-📌 Testing  the app thoroughly to ensure its quality, usability, and performance. <br>
-📌 Releasing the app <br>
-📌 Continuously improve the app's features, design, and functionality based on user <br>
+📌 We have firstly identify the need of children like theri problems in studying <br>
+📌 First of all when user will get to our app ,there will be Google authentication for login. <br>
+📌 User can login by their mail as it is easy to do <br>
+📌 Google authentication is very secured to use <br>
+📌 We have many rhymes for vast variety of choice <br>
+📌 One can read the text and one can also play the text, as for kids reading one is hard so they can play the text also <br>
```

Comment:
One can read the text and also play with it using text-to-speech functionality, as for kids reading one is hard so they can listen to the text as well.